garage/contrib/ros/docker/sawyer/Dockerfile.cpu

```diff
@@ -0,0 +1,177 @@
+FROM ubuntu:16.04
```

Comment:
Please rename this Dockerfile.contrib-ros and put it in docker/

Reply:
Please base Dockerfiles intended to be run interactively on [tini](https://hub.docker.com/r/krallin/ubuntu-tini/)